crates/proto/src/rr/rdata/caa.rs

```diff
@@ -844,20 +844,22 @@ impl fmt::Display for Value {
 
         match self {
             Value::Issuer(name, values) => {
-                match name {
-                    Some(name) => write!(f, "{}", name)?,
-                    None => write!(f, ";")?,
-                }
-
-                if let Some(value) = values.first() {
-                    write!(f, " {}", value)?;
-                    for value in &values[1..] {
+                if name.is_none() && values.is_empty() {
```

Comment:
I think we can simplify to something like this:

```rust
if let Some(name) = name {
    write!(f, "{}", name)?;
}

for value in values {
    write!(f, "; {}", value)?;
}
```

This has slightly different output than what you've got, but I think the output would be compliant with the https://www.rfc-editor.org/rfc/rfc8659.html#name-caa-issue-property syntax.

Reply:
Sorry for taking long to follow up on this.
I was confused because our formatting won't be able to output `;`.
I read the RFC a few more times, and I agree with this change.

One conclusion of this is that `;` and `` (empty issue) should be semantically equal, right?